static/app/views/projectInstall/createProject.tsx

```diff
@@ -283,15 +343,21 @@ export function CreateProject() {
         return;
       }
 
+      let project: Project | undefined = undefined;
+      let customRuleId: string | undefined = undefined;
+      let notificationRuleId: string | undefined = undefined;
```

Comment:
not happy with this, but it will do for now. In a follow-up, we should refactor this using reusable react-query hooks, which would help clean up and improve the whole component code